### PR TITLE
Prep amazon.aws 8.2.0

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Prep amazon.aws 8.2.0

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

several

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,47 @@ amazon.aws Release Notes
...
+
+Release Summary
+---------------
+
+The amazon.aws 8.2.0 release includes a number of bugfixes, some new features and improvements. This releases also introduces a deprecation for the ``amazon.aws.iam_role`` module, where support for creating and deleting IAM instance profiles using the ``create_instance_profile`` and ``delete_instance_profile`` options has been deprecated and will be removed in a release after 2026-05-01.
+
+Minor Changes
+-------------
+
+- cloudwatch_metric_alarm - add  support for `evaluate_low_sample_count_percentile` parameter.
```
Comment (tremble) [inline CHANGELOG.rst:18]:
```suggestion
- cloudwatch_metric_alarm - add  support for ``evaluate_low_sample_count_percentile`` parameter.
```

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -3058,3 +3058,94 @@ releases:
...
+      - 's3_bucket - fixes ``TypeError: cannot unpack non-iterable NoneType object``
+        errors related to bucket versioning, policies, tags or encryption (https://github.com/ansible-collections/amazon.aws/pull/2228).'
+      deprecated_features:
+      - iam_role - support for creating and deleting IAM instance profiles using the
+        ``create_instance_profile`` and ``delete_instance_profile`` options has been
+        deprecated and will be removed in a release after 2026-05-01.  To manage IAM
+        instance profiles the ``amazon.aws.iam_instance_profile`` module can be used
+        instead (https://github.com/ansible-collections/amazon.aws/pull/2221).
+      minor_changes:
+      - cloudwatch_metric_alarm - add  support for `evaluate_low_sample_count_percentile`
```
Comment (tremble) [inline changelogs/changelog.yaml:3086]:
```suggestion
      - cloudwatch_metric_alarm - add  support for ``evaluate_low_sample_count_percentile``
```

**Review by tremble — APPROVED** [review]

**Comment by alinabuzachis** [comment]
recheck

**Review by abikouo — APPROVED** [review]

**Comment by tremble** [comment]
InsufficientInstanceCapacity (ec2_instance) and race condition (kms_key / "changed" status).

Tests passed first time around.

Manually merging

**Pull request merged**